symfony/plugins/orangehrmMaintenancePlugin/lib/form/PurgeAuthenticateForm.php

```diff
@@ -0,0 +1,34 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: administrator
+ * Date: 6/9/18
+ * Time: 11:06 AM
+ */
+class PurgeAuthenticateForm extends sfForm{
+    public function configure() {
```

Comment:
No proper space between function and class .. use PSR2 standards to format the code

Reply:
done